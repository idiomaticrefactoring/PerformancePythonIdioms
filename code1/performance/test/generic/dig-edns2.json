[{"id":8794,"opcode":"QUERY","status":"NOERROR","flags":["qr","rd","ra","ad"],"query_num":1,"answer_num":1,"authority_num":0,"additional_num":1,"opt_pseudosection":{"edns":{"version":0,"flags":[],"udp":512}},"question":{"name":"metebalci.com.","class":"IN","type":"A"},"answer":[{"name":"metebalci.com.","class":"IN","type":"A","ttl":59,"data":"104.198.14.52"}],"query_time":53,"server":"8.8.8.8#53(8.8.8.8)","when":"Fri Apr 16 14:19:58 PDT 2021","rcvd":58,"when_epoch":1618607998,"when_epoch_utc":null},{"id":39319,"opcode":"QUERY","status":"NOERROR","flags":["qr","rd","ra"],"query_num":1,"answer_num":1,"authority_num":0,"additional_num":1,"opt_pseudosection":{"edns":{"version":0,"flags":["do"],"udp":512}},"question":{"name":"google.com.","class":"IN","type":"A"},"answer":[{"name":"google.com.","class":"IN","type":"A","ttl":82,"data":"216.58.194.206"}],"query_time":32,"server":"8.8.8.8#53(8.8.8.8)","when":"Fri Apr 16 14:19:58 PDT 2021","rcvd":55,"when_epoch":1618607998,"when_epoch_utc":null}]
