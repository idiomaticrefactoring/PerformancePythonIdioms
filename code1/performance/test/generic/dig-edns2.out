
; <<>> DiG 9.10.6 <<>> @8.8.8.8 metebalci.com google.com +dnssec
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 8794
;; flags: qr rd ra ad; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;metebalci.com.			IN	A

;; ANSWER SECTION:
metebalci.com.		59	IN	A	104.198.14.52

;; Query time: 53 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Fri Apr 16 14:19:58 PDT 2021
;; MSG SIZE  rcvd: 58

;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 39319
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags: do; udp: 512
;; QUESTION SECTION:
;google.com.			IN	A

;; ANSWER SECTION:
google.com.		82	IN	A	216.58.194.206

;; Query time: 32 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Fri Apr 16 14:19:58 PDT 2021
;; MSG SIZE  rcvd: 55

